release-notes/release-notes52.md

```diff
@@ -82,6 +82,22 @@ For an updated list of {{site.productname}} icons, see: [Editor icon identifiers
 
 The following premium plugins have been updated for {{site.productname}} 5.2.
 
+### Accessibility Checker
```

Comment:
This is missing a couple things if you're concerned? 
- We added the new check for checking duplicate ids
- We improved the checking around decorative images.